vignettes/LPWC.Rmd

```diff
@@ -1,28 +1,17 @@
 ---
 title: "LPWC: Lag Penalized Weighted Correlation for Clustering Short Time Series"
 author: "Thevaa Chandereng & Anthony Gitter"
-date: "`r BiocStyle::doc_date()`"
-package: "`r BiocStyle::2'"
-abstract: >
-
-
-output:
-  rmarkdown::html_document:
-    highlight: pygments
-    toc: true
-    toc_float: true
-    fig_width: 5
-    number_sections: true
+output: rmarkdown::html_vignette
 vignette: >
+  %\VignetteIndexEntry{Do more with dates and times in R}
```

Comment:
@thevaachandereng This should be something about LPWC, right?  I think you used example text.